app/vmselect/graphite/tags_api.go

```diff
@@ -177,18 +178,19 @@ func TagsAutoCompleteValuesHandler(startTime time.Time, w http.ResponseWriter, r
 	if err != nil {
 		return fmt.Errorf("cannot setup tag filters: %w", err)
 	}
+	maxTagKeysPerSearch := flags.GetMaxTagKeysPerSearch()
```

Comment:
I think, it's better to introduce new flags:
`search.maxGraphiteTagValues` and `search.maxGraphiteTags`

 Since, graphite package already has own limits for `search.maxGraphiteSeries`.

Reply:
updated